Jenkinsfile

```diff
@@ -285,6 +286,7 @@ pipeline {
             }
         }
     }
+*/
     stage('Sync after release') {
         when {
            expression { isSuccess == '1' }
```

Comment:
Let's update this condition, so we can sync after release in case we deployed to Stage (for now until we have new Prod)

Reply:
Probably we can update stage('WHAT IS THE NEXT STEP') to have this choice to Sync  after release

Reply:
As per our discussion, let's ignore my comments here, we should be able to sync after release anyway